docs/README.md

```diff
@@ -1,36 +1,49 @@
+# PowSyBl OpenLoadflow documentation
+
 These are the documentation sources for the powsybl-open-loadflow features.
 
 Please keep them up to date with your developments.  
 They are published on [powsybl-open-loadflow.readthedocs.io](http://powsybl-open-loadflow.readthedocs.io/) and pull requests are built and previewed automatically.
 
+## Build the documentation
+
 When modifying the website content, you can easily preview the result on your PC.
 
-**First option - in a terminal, navigate to the root of the project and run the following commands:**
+### From a terminal
 
+Navigate to the root of the project and run the following commands:
+
+Install the requirements the first time:
+~~~bash
+pip install -r requirements.txt
 ~~~
-pip install -r docs/requirements.txt
-sphinx-build -a docs ./build-docs
+Build the documentation:
+~~~bash
+sphinx-build -a . ./_build/html
 ~~~
-
-*Note: if the build fails, try with the `-E` option to clear the cache:*
-
+Or
+~~~bash
+make html
```

Comment:
Another comment from @flo-dup on the `powsybl-core` equivalent PR (powsybl/powsybl-core#3311):

> Those commands give the user a warning as docs/README.md is in the docs folder but is not readthedocs documentation. Should we keep the README.md in this folder?

Reply:
Answer from @alicecaron on the `powsybl-core` PR:

> As discussed, the warning does not stop the documentation to be built and as the README.md in the docs folder is quite big (explaining how to handle the documentation etc...) we can keep it and only refer to it from the main repo README.md

=> OK